confgenerator/built-in-config-windows.yaml

```diff
@@ -18,6 +18,7 @@ metrics:
     mssql:
       type: mssql
       collection_interval: 60s
+      receiver_version: ""
```

Comment:
Nit: the field is already optional and will default to an empty string anyway, so for the sake of keeping the built-in config consistent I'd prefer to leave this out. I assume this was just based on the IIS PR so I'm leaving the same comment there too.

Reply:
@jefferbrecht this appears to be set whenever i run `go test -mod=mod github.com/GoogleCloudPlatform/ops-agent/confgenerator -update_golden -v`

Reply:
IIRC that should be fixable with `yaml:"receiver_version,omitempty"` on `MetricsReceiverMssql.Version`.

Reply:
fixed